components/support/nimbus-fml/src/backends/kotlin/mod.rs

```diff
@@ -0,0 +1,186 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::error::Result;
+use askama::Template;
+
+use crate::intermediate_representation::FeatureManifest;
+use crate::{Config, GenerateStructCmd};
+
+mod gen_structs;
+
+pub(crate) fn generate_struct(
```

Comment:
Feel free to completely ignore this - just jotting down what I'm thinking - I don't know how I feel about the name `generate_struct`, it's mostly the `struct` part that feels weird to me... Not a very constructive thought, I'm not sure what the best alternative is